data/campaigns/Eastern_Invasion/scenarios/09_Castle_in_the_Ice.cfg

```diff
@@ -464,21 +453,17 @@
             [note]
                 description= _ "It is winter, daytime is shorter"
             [/note]
-            {FROSTBITE_NOTE 1}
-            {FROSTBITE_NOTE 2}
-            {FROSTBITE_NOTE 3}
-            {FROSTBITE_NOTE 4}
-            {FROSTBITE_NOTE 5}
-            {FROSTBITE_NOTE 6}
-            {FROSTBITE_NOTE 7}
-            {FROSTBITE_NOTE 8}
-            {FROSTBITE_NOTE 9}
-            {FROSTBITE_NOTE 10}
-            {FROSTBITE_NOTE 11}
-            {FROSTBITE_NOTE 12}
-            {FROSTBITE_NOTE 13}
-            {FROSTBITE_NOTE 14}
-            {FROSTBITE_NOTE 15}
+            [note]
+                #po: The amount is likely between 2 and 10 per turn.
+                #po: With the current balancing it’s the same for every difficulty, starting at 2/turn and rising by 1 every 10 turns.
+                description= _ "Living units take $|frostbite_amount cold damage every turn. Damage increases every 10 turns."
```

Comment:
I realize I'm a little late for this, but… instead of `$|`, might it be better to use `delayed_variable_substitution=yes` in the `[objectives]` tag?